site/cypress/e2e/Cypress-Feature/test_forum.spec.js

```diff
@@ -46,6 +47,18 @@ const removeThread = (title) => {
     cy.get('.thread-left-cont > .thread-list-item').contains(title).should('not.exist');
```

Comment:
As an example here is a place you could have used the `thread-list-item` testid to clean up the Cypress tests

Reply:
I think "clean up" should be in a separate PR as to not clutter what is to be reviewed.

Reply:
I was told to comment this here as a suggestion to Rahul. I agree that a cleanup on another PR would be good.